pages/learn/glossary.md

```diff
@@ -47,6 +47,16 @@ The two are, however, entirely different pieces of the conceptual puzzle, with J
 The JSON format is an open format, with its own [homepage](https://www.json.org/), and specifications published in the [ECMA-404](https://www.ecma-international.org/publications-and-standards/standards/ecma-404/) and [RFC-8259](https://datatracker.ietf.org/doc/html/rfc8259) documents from ECMA and the IETF respectively.
 In particular, it is not managed or developed by the JSON Schema team, who simply make use of the format.
 
+### JSON Hyper-Schema
+
+JSON Hyper-Schema extends JSON Schema, offering a [vocabulary](https://json-schema.org/draft/2019-09/json-schema-hypermedia) to annotate JSON documents with hypermedia controls. This extension facilitates the description of links and actions that can be executed on JSON data, making it a potent tool for developing hypermedia-driven APIs.
+
+The essence of JSON Hyper-Schema lies in its ability to define links and actions that can be executed on JSON data. This is achieved through the use of the links keyword, which allows for the creation of dynamic, interactive data representations. For example, a JSON document representing a blog post might include an "author" property. The JSON Hyper-Schema that describes this document could include a hypermedia control that uses the author's identifier to construct a link to the author's profile. This link is a template that, when applied with the actual data, provides direct access to the author's profile, enhancing the user experience by offering a seamless navigation experience.
+
+This approach not only simplifies the interaction with JSON data but also enhances the overall user experience by providing direct access to related resources without the need for additional requests. It allows developers to create rich, interactive JSON documents that can dynamically adapt to the data they represent, making JSON Hyper-Schema a potent tool for developing hypermedia-driven APIs.
```

Comment:
> that can dynamically adapt to the data they represent

I don't understand what you mean by this.

Reply:
Do I change 
"it allows developers to create rich, interactive JSON documents that can dynamically adapt to the data they represent," 
to this :
"Developers can leverage this capability to create dynamic and interactive JSON documents, allowing them to adapt seamlessly to the evolving nature of the data they represent."

It means : JSON documents can adapt dynamically to changes in the data they represent,

Reply:
> Do I change "it allows developers to create rich, interactive JSON documents that can dynamically adapt to the data they represent," to this : "Developers can leverage this capability to create dynamic and interactive JSON documents, allowing them to adapt seamlessly to the evolving nature of the data they represent."
> 
> It means : JSON documents can adapt dynamically to changes in the data they represent,

I will update this according to your review